superset/assets/package.json

```diff
@@ -17,7 +17,7 @@
     "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production webpack --mode=production --colors --progress",
     "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx . && tslint -c tslint.json ./{src,spec}/**/*.ts{,x}",
     "lint-fix": "eslint --fix --ignore-path=.eslintignore --ext .js,.jsx . && tslint -c tslint.json --fix ./{src,spec}/**/*.ts{,x} && npm run clean-css",
-    "clean-css": "prettier --write src/**/*.{css,less,sass,scss}",
+    "clean-css": "prettier --write 'src/**/*.{css,less,sass,scss}'",
```

Comment:
does a linter catch if css isn't formatted properly with this?

Reply:
Not at this point... was just using it to make life easier and catch myself. Does your Prettier PR address that? Don't want to duplicate effort here :)